src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java

```diff
@@ -1953,6 +1954,14 @@ public String getExternalStatusLabel() {
         return externalStatusLabel;
     }
 
+    public String getLocaleExternalStatusLabel() {
```

Comment:
minor point - could there be a util function instead of three ~near duplicate ones in different classes?

Reply:
Thanks @qqmyers , moved the function to DatasetUtil.java